10.0/extra_requirements.txt

```diff
@@ -1,8 +1,12 @@
 # Extra python dependencies
 algoliasearch==1.17.0
+Adyen==1.2.0
+cachetools==2.1.0
+cerberus==1.2
 boto3==1.7.40
 factur-x==0.3
 invoice2data==0.2.74
+magento==3.0
```

Comment:
hmmm do we need these packages in base setup? :)
was meant to stay in example folder maybe?

Reply:
I have two requirements, the base_requirements and the extra_requirement (this is a new file introduced y this PR). The aim of the extra_requirement is to build a full version of the image with most used python lib (basically OCA python dependency + some extra)
You will have this lib only in the "FULL" image (ex: 10.0-full-latest, 10.0-latest-onbuild). If you do not want to have this lib preinstalled, just use the normal image ex : 10.0-latest or 10.0-latest-onbuild

FULL image can be used in two case:
- fast demo (no need to search all dependency)
- centralized python lib dependency (you want to have the same version of package for the majority of your project, so you always use the FULL version and add missing lib there. For the project where you want to manage the dependency yourself, just use normal image).